src/elm/CompoundComponents/Eth/ConnectedEthWallet.elm

```diff
@@ -569,9 +582,9 @@ chooseWalletView userLanguage isCompoundChain ({ chooseWalletState } as model) =
                          ]
                             ++ headerDescriptions
                             ++ [ div [ class "connect-choices" ]
-                                    ([ connectItemView userLanguage isCompoundChain Metamask
-                                     ]
-                                        ++ lineDivider
+                                     (
+                                           [ connectItemView userLanguage isCompoundChain Metamask]
```

Comment:
We'll need to keep the `lineDivider` right after Metamask and we'll need to insert another `lineDivider` right after the Tally connect item.

Reply:
These items have been added.